README.md

```diff
@@ -1 +1,2 @@
-# RapidRevison
\ No newline at end of file
+# RapidRevison
+## Made with 🖐 by Ayush Singh. 
```

Comment:
Replace 🖐 with :heart: 
Lets revert this PR for a new one

Reply:
Sure

> Replace 🖐 with ❤️
> Lets revert this PR for a new one